charts/argo-cd/values.yaml

```diff
@@ -714,7 +714,15 @@ controller:
     rules:
       # -- Deploy a PrometheusRule for the application controller
       enabled: false
-      # -- PrometheusRule.Spec for the application controller
+      # -- PrometheusRule namespace
+      namespace: "" # "monitoring"
+    
+      selector: {}
+      # prometheus: kube-prometheus
```

Comment:
Does this need to be indented one more to match the others in the file?

Reply:
most of the helm charts that I've looked at (public and private) treat the `#` comment symbol as invisible.  Expecting the consumer to replace the symbol with a space.  Doing it the other way around can lead to confusion in some scenarios.  An example:
```yaml
rootDict:
  key1: value1
# key2: value2
```

With this in mind I see that a lot of the helm chart does not treat the comment as invisible, however there are some exceptions.  [Comment followed by two spaces](https://github.com/argoproj/argo-helm/blob/3a6617ce8f76284776486007cf3279501068cc83/charts/argo-cd/values.yaml#L85) and [Invisible comment](https://github.com/argoproj/argo-helm/blob/3a6617ce8f76284776486007cf3279501068cc83/charts/argo-cd/values.yaml#L91).  This is by no means comprehensive I'm just cherry picking.  The way that I'm used to reading values files tells me that [this ](https://github.com/argoproj/argo-helm/blob/3a6617ce8f76284776486007cf3279501068cc83/charts/argo-cd/values.yaml#L702)is meant to be indented by 4 spaces, however I know that's not the case.  You tell me which way you'd like it formatted and I'll do that.

Reply:
let me know which way you'd like it formatted

Reply:
Whether it's right or wrong, my vote is for consistency with the rest of this project's current choice. Wdyt?

Reply:
alright indented to match the others.